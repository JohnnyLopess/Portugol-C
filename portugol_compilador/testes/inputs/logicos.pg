programa
{
inicio
inteiro x
logico y
logico z
x = 10 * 2
y = x > 15
z = x < 20
escreva("x = 10*2 = 20\n")
escreva("x>15?\n")
escreva(y)
escreva("x>20?\n")
escreva(z)
fim
}