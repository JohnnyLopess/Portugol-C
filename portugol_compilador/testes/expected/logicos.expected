#include <stdio.h>

int main() {
    int x;
    bool y;
    bool z;
    x = 20;
    y = x > 15;
    z = x < 20;
    printf("x = 10*2 = 20\n");
    printf("x>15?\n");
    printf("%d\n", y);
    printf("x>20?\n");
    printf("%d\n", z);
    return 0;
}